### ISSUE TITLE
GDBM_compat Shared Library patch

### ISSUE THREAD

**Issue by BrisingrAerowing** [Alexpux/MINGW-packages#620]
I have created a patch for building gdbm_compat as a shared library:

``` diff
--- compat/Makefile.am.orig 2015-05-10 14:48:32.645958700 -0400
+++ compat/Makefile.am  2015-05-10 14:48:53.183070800 -0400
@@ -50,5 +50,5 @@

 libgdbm_compat_la_SOURCES = $(DBM_CF) $(NDBM_CF)

-libgdbm_compat_la_LDFLAGS = -version-info $(VI_CURRENT):$(VI_REVISION):$(VI_AGE)
+libgdbm_compat_la_LDFLAGS = -version-info $(VI_CURRENT):$(VI_REVISION):$(VI_AGE) -no-undefined
```

(Apply with -p0)

Not a real issue, just something that bothered me a little bit.

**Comment by Alexpux** [comment]
I'm disable build compat library as it break python builds

**Comment by BrisingrAerowing** [comment]
I noticed that after submitting this request.

I wonder what the actual issue with GDBM-compat and python is, though.

**Comment by lazka** [comment]
I've disabled the gdbm module in Python in the meantime:  #3065 #3064

And the compat lib is included since 080bc0b08cd9116f625
